packages/hop-node/src/watchers/SyncWatcher.ts

```diff
@@ -892,8 +892,33 @@ class SyncWatcher extends BaseWatcher {
     }
 
     logger.debug(
-      `looking for transfer ids for transferRootHash ${transferRootHash}`
+      `looking in db for transfer ids for transferRootHash ${transferRootHash}`
     )
+
+    const items = await this.db.transfers.getTransfersWithTransferRootHash(transferRootHash)
+    if (items.length) {
+      const transferIds = items.map(item => item.transferId) as string[]
+      const tree = new MerkleTree(transferIds)
+      const computedTransferRootHash = tree.getHexRoot()
+      if (computedTransferRootHash === transferRootHash) {
+        const transferRootId = this.bridge.getTransferRootId(
+          transferRootHash,
+          totalAmount
+        )
+
```

Comment:
A debug statement here would be very useful for logging. Something like the one below (`found transfer ids for...)  but unique so it is easily searchable.

Reply:
Added in https://github.com/hop-protocol/hop/pull/323/commits/f6326655d2619b29605acce132ee17a7649dc942